libraries/engage/product/components/RelationsSlider/RelationsSliderContent.jsx

```diff
@@ -28,8 +32,9 @@ const RelationsSliderContent = ({ products, getRelations }) => {
   }
 
   return (
-    <Fragment>
+    <div className={styles.container}>
       {!!headline && <h3 className={styles.headline}>{headline}</h3>}
+      {!!showMoreButton && <RelationsSheet limit={100} productId={productId} type={type} />}
```

Comment:
Our PO asked if we could only show the button, when there are really more products available, than we show within the slider. I've checked how the pipeline data is handled within Redux to get an idea how to implement this. 
Right now i only see the option to request one product more, that we want to display within the slider. The crappy thing with the action / selectors is, that the `limit` is part of the hash parameters. So we can only select exactly as much related products as requested.

Reply:
Yes we can. I have implemented that.